### ISSUE TITLE
Can I use this for a reversible AC motor?

### ISSUE THREAD

**Issue by kjjordans** [misan/dcservo#66]
I have a Chinese laser cutter with an AC motor like this that controls the up and down movement of the bed. 

![](https://ae01.alicdn.com/kf/HTB1p1GAXAH0gK0jSZFNq6xMqXXa5/220V-AC-Synchronous-Gear-Motor-68KTYZ-68-KTYZ-28W-Permanent-Magnet-Synchronous-Gear-Motor-220V-2.jpg)

It is controlled by two momentary switches for one forward and one reverse. 
I thought that I could use dcservo with two solid state relays to control the motor. 

![](https://core-electronics.com.au/media/catalog/product/cache/1/image/650x650/fe1bcd18654db18f328c2faaaf3c690a/c/e/ce05347-2.jpg)

Am I out of my mind or do you think that it will work?

**Comment by misan** [comment]
Yes, you can, but you will need to add a variable frequency drive (VFD)
which will make it an expensive proposition.

Alternatively, you could use vESC or ODrive projects if you remove the
capacitor from your motor, as it seems a regular brushless motor. None of
these are cheap either, but they can do the job.

**Comment by kjjordans** [comment]
Thanks for the quick response.
The motor is quite slow so I don’t think I need pwm. Just on and off would be enough. It might be easier to swap the motor for a stepper

**Comment by misan** [comment]
It all depends on the error you can accept. You can try with a dead-band to
prevent constant oscillation (so the motor is not activated until the
output signal is higher than a given threshold).
